app/models/post.rb

```diff
@@ -1,2 +1,17 @@
 class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  has_many :comments, class_name: 'Comment', foreign_key: 'post_id'
+  has_many :likes, class_name: 'Like', foreign_key: 'post_id'
```

Comment:
[OPTIONAL] Kindly remove the class_name present here because the purpose is to allow the naming of a class to be different from what Rails expects. And rails already knows that model names are to be singular